### DISCUSSION TITLE
need to fix error

### DISCUSSION THREAD

**Discussion by udaykumar2104** [medusajs/medusa#6638] [Q&A]
node_modules/type-fest/source/camel-case.d.ts:25:32 - error TS1005: '?' expected.

25  infer FirstWord extends string,
                                  ~

node_modules/type-fest/source/camel-case.d.ts:26:42 - error TS1005: '?' expected.

26  ...infer RemainingWords extends string[],
                                            ~

node_modules/type-fest/source/join.d.ts:59:36 - error TS1005: '?' expected.

59    infer First extends JoinableItem,
                                      ~

node_modules/type-fest/source/join.d.ts:60:49 - error TS1005: '?' expected.

60    ...infer Tail extends readonly JoinableItem[],
                                                   ~

node_modules/type-fest/source/join.d.ts:64:50 - error TS1005: '?' expected.

64     ...infer Head extends readonly JoinableItem[],
                                                    ~

node_modules/type-fest/source/join.d.ts:65:36 - error TS1005: '?' expected.

65     infer Last extends JoinableItem,
                                      ~

Found 6 errors.

from last two days i am getting error

**Comment by sirajlekhak** [comment]
may be you can try this

Check TypeScript Version:
Make sure you're using a version of TypeScript that works well with the type-fest package. You can find the recommended TypeScript version in the package.json file of the type-fest package.

Update Dependencies:
Run `npm update` to ensure all your project's dependencies are up to date. This might fix any compatibility issues.

Check tsconfig.json:
Look at your tsconfig.json file and make sure it's set up correctly. Ensure that the TypeScript version specified in tsconfig.json matches the one you've installed.

Reinstall type-fest:
Sometimes, problems can occur due to a bad installation. Try uninstalling and reinstalling the type-fest package:

bash
```

npm uninstall type-fest
npm install type-fest
```

Check type-fest Version:
Run `npm ls type-fest `to see which version of type-fest you have. This can help identify if there's a mismatch with your TypeScript version.

**Reply by SalahAdDin** [reply]
@udaykumar2104 did you fix it?
